packages/preview/cetz/0.2.2/typst.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "cetz"
+version = "0.2.2"
+compiler = "0.10.0"
+repository = "https://github.com/johannes-wolf/cetz"
+entrypoint = "src/lib.typ"
+authors = [
+  "Johannes Wolf <https://github.com/johannes-wolf>",
+  "fenjalien <https://github.com/fenjalien>"
+]
+license = "Apache-2.0"
+description = "Drawing with Typst made easy, providing an API inspired by TikZ and Processing. Includes modules for plotting, charts and tree layout."
+keywords = [ "draw", "canvas", "plot", "chart", "tree" ]
+exclude = [ "/gallery/*", "manual.pdf", "manual.typ" ]
```

Comment:
You might want to use the opportunitity to add [`categories`](https://github.com/typst/packages/blob/main/CATEGORIES.md).